index.js

```diff
@@ -67,6 +81,31 @@ module.exports = function (src, dest, opts) {
 				resolve(true);
 			});
 
+			var progressEnabled = progress.emitter.listeners('progress').length > 0;
+			if (progressEnabled) {
```

Comment:
With the change above, this can be changed to:
```js
if (eventEmitter.listeners('progress').length > 0) {
```

However I think that optimization isn't worth being included, because [`EventEmitter.prototype.listeners`](https://github.com/nodejs/node/blob/706778a90247937af0af0ae668fabf03e142640b/lib/events.js#L430) creates some overhead, if we have listeners. And [`EventEmitter.prototype.emit`](https://github.com/nodejs/node/blob/706778a90247937af0af0ae668fabf03e142640b/lib/events.js#L173) may not create much, because of run-time optimizations.